packages/demoing-storybook/src/start/transformers/createServeStorybookTransformer.js

```diff
@@ -1,15 +1,53 @@
-module.exports = function createServeStorybookTransformer(assets) {
-  return function serveStorybookTransformer({ url }) {
+const { injectStories } = require('../../shared/injectStories');
+const { createOrderedExports } = require('../../shared/createOrderedExports');
+
+/**
+ * @param {object} args
+ * @param {object} args.assets
+ * @param {string} args.assets.indexHTML
+ * @param {string} args.assets.iframeHTML
+ * @param {string} args.previewImport
+ * @param {string[]} args.storiesPatterns
```

Comment:
would have liked to see more docs
```suggestion
 * @param {string[]} args.storiesPatterns glob patterns of story files
```